src/applications/mhv-medical-records/components/RecordList/AllergyListItems/AllergiesListItem.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StandardAllergyListItem from './StandardAllergyListItem';
+
+const AllergiesListItem = props => {
+  const { record } = props;
+
+  // Since v2 data is now converted to match v1 format exactly,
+  // we can use the same component for both v1 and v2 data
+  return <StandardAllergyListItem record={record} />;
```

Comment:
Any particular reason for breaking the content out to `StandardAllergyListItem` component?